app/modules/intelligence/tools/tool_service.py

```
@@ -69,15 +68,11 @@ def _initialize_tools(self) -> Dict[str, Any]:
             "get_node_neighbours_from_node_id": GetNodeNeighboursFromNodeIdTool(
                 self.db
             ),
+            "ollama_tool": ChatOllama(
```

Ruff (0.8.2)

Undefined name `ChatOllama`

(F821)